core/trino-main/src/main/java/io/trino/operator/TrinoOperatorFactories.java

```diff
@@ -32,9 +32,7 @@
 import static com.google.common.collect.ImmutableList.toImmutableList;
 
 public class TrinoOperatorFactories
```

Comment:
Let's rename this to `OperatorFactories` since the `Trino` prefix doesn't add any value. This will make the diff smaller, too. We could also make this class final.